lib/beaker/shared/repetition.rb

```diff
@@ -7,6 +7,17 @@ def repeat_for seconds, &block
         done = false
         until done or timeout < Time.now do
           done = block.call
+          sleep 1 # do not peg CPU if &block took less than 1 second
```

Comment:
@sbeaulie since the code for `repeat_for_and_wait` method is pretty much copy-pasta from this one, shouldn't this method be reduced to calling that one & passing in `1` as the `wait` parameter?

For a reason why we might want to do that, shouldn't this method also have the `unless done` after the sleep statement? Rather than needing to fix this here, calling the other method will give us that fix in a more maintainable way, no?

Reply:
yes that makes sense, thanks for the suggestion